core/src/commonMain/kotlin/com/powersync/db/schema/Table.kt

```diff
@@ -33,6 +37,22 @@ public data class Table constructor(
      * Override the name for the view
      */
     private val viewNameOverride: String? = null,
+    /**
+     *  Whether to add a hidden `_metadata` column that will be enabled for updates to attach custom
+     *  information about writes that will be reported through [CrudEntry.metadata].
+     */
+    val includeMetadata: Boolean = false,
```

Comment:
This is just me blurbing out my initial thoughts on the naming of these variables.

The `include` here takes my brain a few cycles to understand what it means. The comment does explain it pretty well though. Maybe we could call this `trackMetadata`? Perhaps the same for `includeOld` -> `trackPreviousValues`?